lib/src/widgets/switch/ds_switch.widget.dart

```diff
@@ -0,0 +1,94 @@
+import 'package:blip_ds/blip_ds.dart';
+import 'package:flutter/material.dart';
+
+/// Animation duration for [DSSwitch].
+const Duration _animationDuration = Duration(milliseconds: 400);
+
+class DSSwitch extends StatelessWidget {
```

Comment:
could create a widget base, to share as common properties between ds_switch_tile and ds_switch
example:

`class DSSwitchBase extends StatelessWidget {
  /// Function callback to use when the [onChanged].
  final ValueChanged<bool> onChanged;

  /// Enables or disables the widget by changing the color of the switch.
  final bool isEnabled;

  /// Referring to [isActive] true ou false.
  final bool isActive;

  /// Referring to [inactiveColor] for  [isActive] false,
  final Color inactiveColor = DSColors.neutralMediumSilver;

  /// Color for background of the switch widget when [isActive] is true.
  final Color activeColor = DSColors.primaryNight;

  DSSwitchBase(
      {Key? key,
      required this.onChanged,
      required this.isEnabled,
      required this.isActive})
      : super(key: key);

  @override
  Widget build(BuildContext context) {
    return Container();
  }
}`

`class DSSwitch extends DSSwitchBase {
  /// Creates a customizable switch for use alone or in conjunction with ListTile
  ///
  /// Set the active and inactive colors using the parameters [activeColor] and [inactiveColor], and use [isActive] to turn the switch on and off.
  DSSwitch({
    Key? key,
    required super.onChanged,
    required super.isActive,
    super.isEnabled = true,
  }) : super(key: key);

............`

Reply:
Done